fecfile/cache.py

```diff
@@ -38,12 +38,11 @@ def getMapping_from_regex(mappings, form, version):
 
 def getMapping(mappings, form, version):
     """ Tries to find the mapping from cache before looking it up w regex """
-    key = MAPPING_CACHE_KEY % (form, version)
-    try:
-        mapping = MAPPING_CACHE[key]
-    except KeyError:
-        mapping = getMapping_from_regex(mappings, form, version)
-        MAPPING_CACHE[key] = mapping
+    key = (form, version)
+    if key in MAPPING_CACHE:
+        return MAPPING_CACHE[key]
```

Comment:
I wonder if doing a `MAPPING_CACHE.get(key)` once, rather than doing an `in` and then a lookup would be faster?

Reply:
Basically we want to return if it's in the cache, and if not we want to do the lookup. Could you clarify how `.get` would work in this scenario?

Reply:
```
mapping = MAPPING_CACHE.get(key)
if mapping:
    return mapping
```

https://wiki.python.org/moin/TimeComplexity

Reply:
Actually upon careful reading of this page it prob won't make a difference because for dict retrievals they both look like `O(1)`, but I do think with `.get()` can you reduce it to one `O(1)` instead of two (the `in` on l42 and the `[]` on l43`

Reply:
Ah yea that makes complete sense. I was thinking get would be used in a different way... implemented!